src/App.jsx

```diff
@@ -14,45 +20,94 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+export const App = () => {
+  const [select, setSelect] = useState('');
+  const [isReversed, setReversed] = useState();
+
+  const sortMethod = () => {
+    let res = [...goodsFromServer];
+
+    if (select === ALPHABETICALLY) {
+      res = [...goodsFromServer].sort((a, b) => a.localeCompare(b));
+    }
+
+    if (select === LENGTH) {
+      res = [...goodsFromServer].sort((a, b) => a.length - b.length);
+    }
+
+    if (select === RESET) {
+      return goodsFromServer;
+    }
+
+    if (isReversed) {
+      res = res.reverse();
+    }
+
+    return res;
+  };
 
-    <ul>
-      <li data-cy="Good">Dumplings</li>
-      <li data-cy="Good">Carrot</li>
-      <li data-cy="Good">Eggs</li>
-      <li data-cy="Good">Ice cream</li>
-      <li data-cy="Good">Apple</li>
-      <li data-cy="Good">...</li>
-    </ul>
-  </div>
-);
+  const Reset = () => {
+    if (select !== '' && !isReversed) {
```

Comment:
check https://mate-academy.github.io/react_list-of-goods/
you don't need to hide the button when !isReversed